### PR TITLE
[2021.2] Universal: Updating various assets and files

### PR DESCRIPTION
**Pull request by ellioman**
# Purpose of this PR
When opening one of the Split Projects, with Unity 2021.2, you will get a script updater which changes various files and assets. This PR is committing those changes to the repo.

# Yamato
https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/universal%252Fupdate-projects-and-files

### CODE REVIEW

File: com.unity.render-pipelines.universal/Runtime/2D/Light2DManager.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 using System.Collections.Generic;
 
 #if UNITY_EDITOR
-using UnityEditor.Experimental.SceneManagement;
+
 #endif
```
Comment (svens-unity) [inline com.unity.render-pipelines.universal/Runtime/2D/Light2DManager.cs:5]: Remove ifdefs

Comment (ellioman) [reply com.unity.render-pipelines.universal/Runtime/2D/Light2DManager.cs]: Done! 👍

File: TestProjects/UniversalGraphicsTest_Terrain/Assets/Scenes/140_SSAO/Renderers/SSAO_DeferredRenderer.asset [hunk 2]
```diff
@@ -41,8 +40,8 @@ MonoBehaviour:
     failOperation: 0
     zFailOperation: 0
   m_ShadowTransparentReceive: 1
-  m_RenderingMode: 1
```
Comment (phi-lira) [inline TestProjects/UniversalGraphicsTest_Terrain/Assets/Scenes/140_SSAO/Renderers/SSAO_DeferredRenderer.asset:44]: Is this correct? Seems like m_RenderingMode = 0 is changing to forward renderer?

File: TestProjects/UniversalGraphicsTest_Terrain/Assets/Scenes/140_SSAO/Renderers/SSAO_DepthNormalRenderer.asset [hunk 3]
```diff
@@ -31,7 +32,7 @@ MonoBehaviour:
   m_GameObject: {fileID: 0}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: de640fe3d0db1804a85f9fc8f5cadab6, type: 3}
```
Comment (phi-lira) [inline TestProjects/UniversalGraphicsTest_Terrain/Assets/Scenes/140_SSAO/Renderers/SSAO_DepthNormalRenderer.asset:34]: Lot's of GUId changes, is this expected?

**Review by phi-lira — APPROVED** [review]

**Review by hdb-unity — APPROVED** [review]
The PostPro project passes in playmode when locally run in min Unity version.
From discussion, seems like guid changes are expected.

File: TestProjects/UniversalGraphicsTest_Terrain/Assets/Scenes/140_SSAO/Renderers/SSAO_DepthNormalRenderer.asset (hunk 3 above)
Comment (ellioman) [reply TestProjects/UniversalGraphicsTest_Terrain/Assets/Scenes/140_SSAO/Renderers/SSAO_DepthNormalRenderer.asset:34]: Yes, ForwardRenderer -> UniversalRenderer

**Pull request merged**